### PR TITLE
Circom Hash Checker Tutorial

### PR DESCRIPTION
**Pull request by ioedeveloper**

### CODE REVIEW

File: CircomHashChecker/step-3/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+### Code Breakdown
+
+#### Pragma and Includes:
+- `pragma circom 2.0.0;` specifies the Circom version.
+- `include "circomlib/circuits/poseidon.circom";` fetch and includes the Poseidon hash function from [CircomLib](https://github.com/iden3/circomlib).
+
+#### `CalculateHash` Template:
+- Defines inputs `value1`, `value2`, `value3`, `value4`.
+- Uses the `Poseidon` hash function to compute a hash of these values.
+- Uses `assert(hash == calculatedHash);` to ensure the provided hash matches the calculated hash.
```
Comment (yann300) [inline CircomHashChecker/step-3/README.md:14]: this part is in the `HashChecker` function

Comment (yann300) [reply CircomHashChecker/step-3/README.md:14]: I mean it's not in the `CalculateHash` template as stated above.

Comment (ioedeveloper) [reply CircomHashChecker/step-3/README.md:14]: It is the template. Scroll down

Comment (yann300) [reply CircomHashChecker/step-3/README.md:14]: I mean `assert(hash == calculatedHash);` is not in the `CalculateHash` template, it's in the `HashChecker` template.

**Pull request merged**